client/src/auth_config.js

```diff
@@ -0,0 +1,6 @@
+const config = {
+  domain: 'greenocean.auth0.com',
+  clientId: '05RvxwAP7dSW5I9uPHxP6m7hVKHoIjS3'
```

Comment:
Should this be in GitHub?

Reply:
Nope! I added it to gitignore but the file extension was wrong so it was being included.